<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_page</class>
 <widget class="QMainWindow" name="main_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>201</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_gen_gif">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Gif</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_filein">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browse_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>35</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dbl_start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000000000000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dbl_end">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100000000000000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>end</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Video File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_gif">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of GIF</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spn_fps">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>fps</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dbl_scale">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>62</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spn_fuzz">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>fuzz</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
